packages/teleport/src/SideNav/SideNavExternalLink.tsx

```diff
@@ -0,0 +1,97 @@
+/*
+Copyright 2022 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import React from 'react';
+import styled from 'styled-components';
+import defaultTheme from 'design/theme';
+import { color } from 'design/system';
+
+import { ArrowForward } from 'design/Icon';
+
+import SideNavItemIcon from './SideNavItemIcon';
+
+const fromTheme = ({ $nested = false, theme = defaultTheme }) => {
+  const css = {
+    position: 'relative',
+    fontSize: '12px',
+    fontWeight: theme.regular,
+    fontFamily: theme.font,
+    paddingLeft: theme.space[9] + 'px',
+    paddingRight: theme.space[2] + 'px',
```

Comment:
unless i'm missing something, this file looks almost identical to `SideNavItem.tsx`. If all we wanted was to change the padding (and type of element used which we can use the `as` prop) perhaps we can add `${space}` from design/system so we can pass in our own padding value? (OR, i'm pretty sure we can just use the inline `css` prop as well) 

like this;

```ts
import { color, space } from 'design/system';

const SideNavItem = styled.div`
  box-sizing: border-box;
  display: flex;
  align-items: center;
  justify-content: flex-start;
  border: none;
  border-left: 4px solid transparent;
  cursor: pointer;
  outline: none;
  text-decoration: none;
  width: 100%;
  line-height: 24px;
  ${fromTheme}  // default settings
  ${color}               // customize color
  ${space}            // customize spacing
`;

```
then we can do

```ts
// design system
const SideNavExternalLink = ({ children, href, icon }) => {
  return (
   // note the `as` prop, we can instruct this comp to be something other than a default `div`
    <SideNavItem as="a" href={href} target="_blank" pr={2}> 
      <SideNavItemIcon as={icon} />
      {children}
      <SideNavItemIcon
        css={{ marginLeft: 'auto', transform: 'rotate(-45deg)' }}
        as={ArrowForward}
      />
    </SideNavItem>
  );
};

// inline css (this is what i would do, more simple for one change)
const SideNavExternalLink = ({ children, href, icon }) => {
  return (
    // note the `as` prop, we can instruct this comp to be something other than a default `div`
    <SideNavItem as="a" href={href} target="_blank" css={{paddingRight: `${theme.space[2]}px`}}>  
      <SideNavItemIcon as={icon} />
      {children}
      <SideNavItemIcon
        css={{ marginLeft: 'auto', transform: 'rotate(-45deg)' }}
        as={ArrowForward}
      />
    </SideNavItem>
  );
};

// another way to define inline css, really helpful if you need to use pseudo classes (eg: :hover { stuff })
const SideNavExternalLink = ({ children, href, icon }) => {
  return (
    // note the `as` prop, we can instruct this comp to be something other than a default `div`
    <SideNavItem as="a" href={href} target="_blank" css={`padding-right: ${theme.space[2]}px}>  
      <SideNavItemIcon as={icon} />
      {children}
      <SideNavItemIcon
        css={{ marginLeft: 'auto', transform: 'rotate(-45deg)' }}
        as={ArrowForward}
      />
    </SideNavItem>
  );
};
```

Reply:
I just copied it at first and forgot to refactor it back. I used your inline CSS approach, looks great now, thank you very much!